auth-web/src/views/auth/staff/StaffDashboardView.vue

```diff
@@ -376,7 +380,9 @@ export default defineComponent({
       searchIdentifier: '',
       canSearchFAS: computed((): boolean => currentUser.value?.roles?.includes(Role.FasSearch)),
       canViewAccounts: computed((): boolean => currentUser.value?.roles?.includes(Role.StaffViewAccounts)),
-      canViewAllTransactions: computed((): boolean => currentUser.value?.roles?.includes(Role.ViewAllTransactions)),
+      canViewAllTransactions: computed((): boolean => currentUser.value?.roles?.some(
+        role => role === Role.ViewAllTransactions || role === Role.ContactCentreStaff
```

Comment:
I think we should grant the new group the ability to view all transactions in this case then. I have applied this in dev. We shouldn't need to code for contact centre staff here.

Let me know if this works, if not we can dig into it a bit more